File: lib/Models/Shows.dart
class Shows {
  final String id;
    final String showPoster;
  final String showName;
  final String showDescription;

  Shows._(this.id,this.showPoster,this.showName,this.showDescription);

  factory Shows.fromJson(Map json) {
    final id = json[''].replaceAll('ObjectId(\"', '').replaceAll('\")', '');
    final showPoster = json['showPoster'];
    final showName = json['showName'];
    final showDescription = json['showDescription'];

    return Shows._(id,showPoster,showName,showDescription);
  }
}

File: AppAPI/Models/Shows.js
const mongoose = require('mongoose');
const Schema = mongoose.Schema;

const Shows = new Schema({
    showPoster:{
        type: String,
        required: true
    },
    showName: 
    {
        type: String,
        required: true
    },
    showDescription: 
    {
        type: String,
        required: true
    }
});
mongoose.model('Shows',Shows);